docs/Toy/Toy_Ch0.ipynb

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
<p>The unknown assembler directives are required for our Verilator model and not supported/ignored yet by riscemu, hopefully a temporary situation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xdslproject/xdsl/pull/1554/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>